src/embedding.jl

```diff
@@ -0,0 +1,253 @@
+# Make sure two things :: correct setup & compatible mapping function
```

Comment:
Please add a comment for each function

Reply:
This is taken care of in the most recent commit.